app/src/main/res/layout/pbm_shortcut_popup.xml

```diff
@@ -15,7 +15,7 @@
         android:layout_marginEnd="@dimen/cfr_triangle_margin_edge"
         android:importantForAccessibility="no"
         app:srcCompat="@drawable/ic_cfr_triangle"
-        app:tint="#7542E5" />
+        app:tint="@color/fx_mobile_layer_color_gradient_end" />
```

Comment:
This is an intentional change since the triangle is on the gradient end side.

<img width="588" alt="Screen Shot 2022-02-24 at 1 14 49 PM" src="https://user-images.githubusercontent.com/1190888/155583487-4447425b-8ea7-4161-8b70-d48f8cc590f4.png">